frontend/src/container/withActivity.jsx

```diff
@@ -15,7 +16,8 @@ import {
 } from '../util/activity.js'
 import {
   FETCH_CONFIG,
-  handleClickCopyLink
+  handleClickCopyLink,
+  CONTENT_NAMESPACE
```

Comment:
I don't think this is necessary nor a rule, but it might be good to keep constants grouped. Feel free to ignore this suggestion.

```suggestion
  CONTENT_NAMESPACE,
  handleClickCopyLink
```